sklearn/kernel_ridge.py

```diff
@@ -83,9 +83,9 @@ class KernelRidge(BaseEstimator, RegressorMixin):
 
     See also
     --------
-    Ridge
+    :func:`sklearn.linear_model.Ridge`:
```

Comment:
I think numpydoc should interpret this without func. But you might need to prefix with linear_model etc